asm/non_matchings/code_80091750/func_8009F5E0.s

```diff
@@ -597,26 +597,26 @@ glabel L8009FD90
 glabel L8009FE9C
 /* 0A0A9C 8009FE9C 3C028019 */  lui   $v0, %hi(D_8018EDF3) # $v0, 0x8019
 /* 0A0AA0 8009FEA0 8042EDF3 */  lb    $v0, %lo(D_8018EDF3)($v0)
-/* 0A0AA4 8009FEA4 3C03800F */  lui   $v1, %hi(D_800E86AB) # 0x800f
-/* 0A0AA8 8009FEA8 3C06800F */  lui   $a2, %hi(D_800E86AD) # 0x800f
+/* 0A0AA4 8009FEA4 3C03800F */  lui   $v1, %hi(gGameTypeRowSelectionForNumPlayers) # 0x800f
+/* 0A0AA8 8009FEA8 3C06800F */  lui   $a2, %hi(gGameTypeSubMenuRowSelectionForNumPlayers) # 0x800f
 /* 0A0AAC 8009FEAC 00621821 */  addu  $v1, $v1, $v0
-/* 0A0AB0 8009FEB0 806386AB */  lb    $v1, %lo(D_800E86AB)($v1) # -0x7955($v1)
+/* 0A0AB0 8009FEB0 806386AB */  lb    $v1, %lo(gGameTypeRowSelectionForNumPlayers)($v1) # -0x7955($v1)
```

Comment:
Open to suggestions for these names.

gGameTypeRowSelectionForNumPlayers is an array storing which row is selected for each of 1P Game, 2P Game, etc. The value stored is an index, not the actual value of the game type. (e.g. gGameTypeRowSelectionForNumPlayers[3] == 0 means versus is selected in the 3P game menu).

gGameModeFromNumPlayersAndGameTypeRowSelection is a 2D array storing what game type each row of the 1P Game, 2P Game, ... menus actually corresponds to. So gGameModeFromNumPlayersAndGameTypeRowSelection[3][0] == 2 (Versus).

gGameTypeSubMenuRowSelectionForNumPlayers is a 2D array storing what row is chosen in the submenu for each game type in each 1P Game, 2P Game, ... menu.
E.g. gGameTypeSubMenuRowSelectionForNumPlayers[1][0] == 2 means 150CC mode in 1P Game, Grand Prix is selected. gGameTypeSubMenuRowSelectionForNumPlayers[2][1] == 0 means 50 CC mode in 2P Game, Versus is selected.

Reply:
That's pretty reasonable. I'm sort of confused in how `GameType` differs from `GameMode`.

Reply:
Ah... they're the same. I'll use `GameMode` to keep things consistent